### ISSUE TITLE
Return is invisible, not `TRUE` 

### ISSUE THREAD

**Issue by damianooldoni** [frictionlessdata/frictionless-r#145]
It seems to me that https://github.com/frictionlessdata/frictionless-r/blob/7dba18d98afe46480367ef2f878160efecb7e629/R/check_package.R#L7 is not correct. The `check_package()` returns nothing if no errrors are found.

**Comment by peterdesmet** [comment]
Indeed, because the last `assertthat::assert_that()` will return TRUE if passing.
